v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -798,6 +798,22 @@ This might be caused by using hardhat_reset and loadFixture calls in a testcase.
       websiteTitle: "Error while reverting snapshot",
       websiteDescription: "Error while reverting snapshot",
     },
+    CAN_ONLY_BE_USED_WITH_HARDHAT_NETWORK: {
+      number: 914,
+      messageTemplate: `This helper can only be used with the Hardhat Network. You are connected to "{networkName}".`,
+      websiteTitle:
+        "Hardhat network helpers can only be used with the Hardhat Network",
+      websiteDescription:
+        "Hardhat network helpers can only be used with the Hardhat Network",
+    },
+    CAN_ONLY_BE_USED_WITH_HARDHAT_NETWORK_VERSIONED: {
+      number: 915,
+      messageTemplate: `This helper can only be used with the Hardhat Network. You are connected to "{networkName}", whose identifier is "{version}".`,
+      websiteTitle:
+        "Hardhat network helpers can only be used with the Hardhat Network",
```

Comment:
Good question! I thought the same but my assumptions was that as long as the keys are not the same (which is impossible because TS would complain) we should be safe.

But to be even safer I changed the message [here](https://github.com/NomicFoundation/hardhat/pull/6010/commits/1dde1f173b64ec11e45edc490d7d7b0ff4d3a8c7)

Reply:
I think your take is right, but it wouldn't hurt to use a slightly different title.